### PR TITLE
[plugin.program.AML] 0.9.12

### PR DESCRIPTION
**Pull request by Wintermute0110**
### Description

This is a minor bug-fix release. Fixed parsing of History.DAT 2.17. New infolabel `$INFO[ListItem.Property(history)]`

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.program.AML/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.program.AML" name="Advanced MAME Launcher" version="0.9.11" provider-name="Wintermute0110">
+<addon id="plugin.program.AML" name="Advanced MAME Launcher" version="0.9.12" provider-name="Wintermute0110">
```
Comment (enen92) [inline plugin.program.AML/addon.xml]: would be cool if you could adjust the `license` element to use a spdx identifier: https://spdx.org/licenses/

File: plugin.program.AML/changelog.txt [hunk 2]
```diff
@@ -18,6 +18,12 @@ WIP      [CORE] AML/AEL must report the aspect ratio of artwork to the skin.
 
 [B]AML ideas and planned features (read to implement)[/B]
 
+WIP      [CORE] Implement the "Read-only launchers". ROLs are XML files that include the
```
Comment (enen92) [inline plugin.program.AML/changelog.txt]: this file is not needed in submissions, it'd be nice if you could drop it in the future. It is not parsed by kodi anymore in favor of `<news>`

**Review by enen92 — APPROVED** [review]

File: plugin.program.AML/addon.xml (hunk 1 above)
Comment (Wintermute0110) [reply plugin.program.AML/addon.xml]: Thank for the advice, I already changed it in master. By the way, the example given in [Kodi Wiki Addon](https://kodi.wiki/view/Addon.xml) in the `<license>` is deprecated according to the list you posted and must be updated.

File: plugin.program.AML/changelog.txt (hunk 2 above)
Comment (Wintermute0110) [reply plugin.program.AML/changelog.txt]: I know. However, AML is a very atypical addon due to its huge size and many features as an emulator frontend. Most of the times the `<news>` tag is insufficient to convey all the information of the new version. I think the best way is to include the `changelog` file and refer to it in the `<news>` tag.

**Pull request merged**